.github/workflows/ci.yml

```diff
@@ -35,11 +35,8 @@ jobs:
           pixi-version: v0.39.0
           cache: true
           environments: lint
-      - name: Run Pylint, Mypy & Pyright
-        run: |
-          pixi run -e lint pylint
```

Comment:
Redundant and incompatible with ruff

Reply:
Pylint is also included in ruff, so why would it be incompatible?

Reply:
there are many pylint rules that ruff hasn't implemented: https://github.com/astral-sh/ruff/issues/970

Reply:
A simple example was 
```python
class at:
```
pylint complains that classes need to be capitalized;
but if you add a `# noqa` to silene pylint, the ruff fails with an "unnecessary noqa" error

Reply:
the way to silence pylint is `  # pylint: disable=invalid-name`